### ISSUE TITLE
Cards Left

### ISSUE THREAD

**Issue by maysw** [cs361-W16/Group11-2#7]
Issue: Implementing Cards Left

I am having issues implementing a box indicating how many cards are left in the game. I have the interface created but I am having issues passing the value to the page and having it update every time cards are dealt or removed

**Comment by matsuish** [comment]
I think cards left part is similar to my part, so it could be helpful to introduce what I have done.

1.I added member valuable "score" in Game class. It is updated when remove() is called. If it's cards left, it should be updated when deal() is called.
2.I added HTML tags 
    <tr class="score">
        <td>Your Score:</td>
        <td id="score"></td>
    </tr>
in table in AcesUp.fit.html. 
By declaring class, I can write font's size and color specification in css file.
By declaring id, I javascript function can find where they should insert score.
3.In javascript part in AcesUp.fit.html, there is display() function which is called whenever game state is changed. I added 
$('#score').html(game.score);
into the display() function. It means to where the id is "score", insert member valuable score in a game class instance.
when any buttons are pushed, display() function is called, so score is updated every time.

that's all I have done. hopefully this could be helpful for your part. But I don't know how to show valuable in a box, sorry.
